### PR TITLE
BUGFIX: Reload document after publish

### PR DESCRIPTION
**Pull request by grebaldi**
Fixes: #1060

### CODE REVIEW

**Comment by dimaip** [comment]
Hmm put that will... reload the document? Do we want that? Is there a way to prevent that? In the old UI we didn't do that.
Maybe we can send this info via a feedback?

**Comment by grebaldi** [comment]
This will be very complicated to figure out on server side... Maybe we should add a condition, to only reload, if there's no `previewUrl` present.

**Comment by dimaip** [comment]
That could be an option, though it's such a pity to reload the whole thing just for the sake of this preview url... Maybe we can get it from src by stripping context?

**Comment by skurfuerst** [comment]
@dimaip @grebaldi what's the status here? :)

**Comment by grebaldi** [comment]
@skurfuerst This feature is not mission critical, but I will look for a solution that doesn't require a full reload. 

I'll add a `WIP` prefix to the PR.

**Comment by dimaip** [comment]
I'm quite surprised, but all of our editors seem to claim this bug *is* mission critical for them...
@grebaldi could you still find time to think it through and fix it in these two months? I think you understand the topic best.

**Comment by dfeyer** [comment]
I think it's fine if we merge this, and create an issue to improve the reloading on a separate PR.

**Comment by grebaldi** [comment]
I'm afraid, I have to disagree. The behavior introduced by this PR pretty much breaks the `auto-publish` feature. 

I'll try to put some thought on a better way to solve this.

**Comment by dimaip** [comment]
Yep, no preview url is a lesser evil. But would be great to fix before the 1.0!

**Comment by grebaldi** [comment]
I updated the PR, the page is no longer reloaded. The preview url gets updated independently, if the current content canvas document node is amongst the recently published nodes. 

I also disabled the preview button, if there's no preview url - before it has been disabled visually, but actually opened the UI in a new tab on click.

File: packages/neos-ui/src/Sagas/Publish/index.js [hunk 1]
```diff
@@ -21,6 +22,19 @@ function * watchPublish() {
                 const feedback = yield call(publish, nodeContextPaths, targetWorkspaceName);
                 yield put(actions.UI.Remote.finishPublishing());
                 yield put(actions.ServerFeedback.handleServerFeedback(feedback));
+
+                //
+                // Reload the current content canvas document node, to retrieve a new preview uri,
```
Comment (dimaip) [inline packages/neos-ui/src/Sagas/Publish/index.js]: Update comment

**Pull request merged**